bin/prepare_content

```diff
@@ -146,13 +152,17 @@ csv_data.each do |row|
   #  or that match exacatly but are in a sub-directory (as indicated by having a path separator, e.g. a "/" right before the filename)
   # e.g. if you are looking for a file called "test.csv", this will match "test", "test.csv", "test.jpg", "dir/test.csv", "dir/test", but NOT "0test", or "dir/0test.jpg"
   files_found = files_to_search.grep(/((.+\/{1}#{filename})|(^#{filename}))\.\S+/i)
-  # if found, symlink files that match
+  # if found, copies or symlinks files that match
   files_found.each do |input_file|
     input_filename = File.basename(input_file)
-    message = "found #{input_file}, symlink to object folder #{object_folder}"
+    message = "found #{input_file}, #{action} to object folder #{object_folder}"
     output_file_full_path = no_object_folders ? File.join(staging_folder, input_filename) : File.join(object_folder, input_filename)
     input_file_full_path = Pathname.new(File.join(base_content_folder, input_file)).cleanpath(true).to_s
-    FileUtils.ln_s(input_file_full_path, output_file_full_path, force: true) unless report || File.exist?(output_file_full_path)
+    if symlink
+      FileUtils.ln_s(input_file_full_path, output_file_full_path, force: true) unless report || File.exist?(output_file_full_path)
```

Comment:
Can you put the unless above instead of repeating?

Reply:
Yes, good catch - relic from copy and paste.  Changed it to be more succinct.